project-base/storefront/components/Pages/Order/ContactInformation/FormBlocks/ContactInformationSendOrderButton.tsx

```diff
@@ -44,7 +44,17 @@ export const ContactInformationSendOrderButton: FC = () => {
                     }}
                 />
             </p>
-
+            {settingsData?.settings?.heurekaEnabled && (
+                <CheckboxControlled
+                    control={formProviderMethods.control}
+                    formName={formMeta.formName}
+                    name={formMeta.fields.isWithoutHeurekaAgreement.name}
+                    render={(checkbox) => <ChoiceFormLine>{checkbox}</ChoiceFormLine>}
+                    checkboxProps={{
+                        label: formMeta.fields.isWithoutHeurekaAgreement.label,
+                    }}
+                />
+            )}
```

Comment:
Why was it moved here?

Reply:
Because we want to have all checkboxes in one place